subnet/subnet_contract.go

```diff
@@ -102,6 +149,11 @@ func Get(id []byte) []byte {
 
 // Delete deletes subnet with the specified id.
 func Delete(id []byte) {
+	// V2 format check
+	if len(id) != subnetIDSize {
+		panic("delete: " + ErrInvalidSubnetID)
+	}
+
 	ctx := storage.GetContext()
 	key := append([]byte{ownerPrefix}, id...)
```

Comment:
`Delete` should remove all subnet related data. Process all new prefixes.

Reply:
do you know how to delete all records using only the prefix? or I need to iterate using prefix and then delete them one by one?

Reply:
> or I need to iterate using prefix and then delete them one by one?

I think this is the way.

Reply:
https://github.com/nspcc-dev/neofs-contract/pull/174/commits/e415cef5791dd87642f165c294e432b4e3d580ca

how about that?